templates/pet-insurance/sections/final-price-consent.js

```diff
@@ -12,19 +12,21 @@ export default (name, { estimatedPrice = {}, finalPrice = {} }, skip, sdk) => {
     if (finalValue > estimatedValue) {
         //template to just display on modal
         const template = html`
-            <p>The final price has changed. and will be:</p>
-            <b class="large">${templates.priceDisplay(finalPrice.price)}</b>
-            <div class="section__actions field field-set">
-                <button type="button" class="button button--right button--secondary" id="cancel-btn">Cancel</button>
-                <button type="button" class="button button--right button--primary" id="submit-btn-${name}">Confirm and pay</button>
+            <p class="dim">The final price has changed. and will be:</p>
```

Comment:
Not from this PR, but this sentence looks weird.

Reply:
true, the dot shouldn't be there